src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,7 @@ public CommandResult execute() {
         }
     }
 
+    public boolean isMutating() {
```

Comment:
Better to put `@Override` annotation so that it's clearer for other programmers.